docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -209,6 +209,8 @@ Metriport makes it easy to share this data back with the networks - you can:
 - Upload FHIR data using our [Create Patient's Consolidated Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 - Upload binary documents using the [Upload Document](/medical-api/api-reference/document/post-upload-url) endpoint.
 
+For more details on how to share data back, see the [Shareback](/medical-api/more-info/shareback) guide.
```

Comment:
we agreed to call this data contribution previously - please update the naming/working accordingly in the docs and I'll take another look